spec/jobs/document_uploads/pdf_conversion_job_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe DocumentUploads::PdfConversionJob do
+  describe '#perform' do
+    before(:all) do
+      @file_path = Rails.root.join('spec/fixtures/files/pdf_conversion.jpg')
```

Comment:
I think it's supposed to be 
```
@file_path = Rails.root.join('spec', 'fixtures', 'files', 'pdf_conversion.jpg')
```

in the unlikely even that this spec is run on an OS where `/` is not the default separator